4.2/init.sh

```diff
@@ -4,7 +4,7 @@ OSMFILE=${PROJECT_DIR}/data.osm.pbf
 
 if [ "$IMPORT_WIKIPEDIA" = "true" ]; then
   echo "Downloading Wikipedia importance dump"
-  curl https://nominatim.org/data/wikimedia-importance.sql.gz -L -o ${PROJECT_DIR}/wikimedia-importance.sql.gz
+  curl -A {$USER_AGENT} https://nominatim.org/data/wikimedia-importance.sql.gz -L -o ${PROJECT_DIR}/wikimedia-importance.sql.gz
```

Comment:
Was that meant to be `-A ${USER_AGENT}`? Same below.

Reply:
Oh yes ... thanks for the catch!